### PR TITLE
[docs] add constraint program formulation of Sudoku

### PR DESCRIPTION
**Pull request by odow**
Part of #2227 

HiGHS failure with presolve on is https://github.com/ERGO-Code/HiGHS/issues/907

Preview: https://jump.dev/JuMP.jl/previews/PR3014/tutorials/linear/sudoku/#Constraint-programming-formulation

### CODE REVIEW

File: docs/Project.toml [hunk 1]
```diff
@@ -5,6 +5,7 @@ Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 GLPK = "60bf3e95-4087-53dc-ae20-288a0d20c6a6"
 HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 HiGHS = "87dc4568-4c63-4d18-b0c0-bb2238e4078b"
+HiGHS_jll = "8fd58aa0-07eb-5a78-9b36-339c94fd15ea"
```
Comment (blegat) [inline docs/Project.toml:8]: Why is this added ?

File: docs/Project.toml [hunk 2]
```diff
@@ -23,14 +24,14 @@ CSV = "0.10"
 DataFrames = "1"
 Documenter = "0.27.9"
 GLPK = "=1.0.1"
-HTTP = "0.9"
+HTTP = "0.9, 1"
```
Comment (blegat) [inline docs/Project.toml:27]: Do we need to keep `0.9` ?

File: docs/Project.toml (hunk 1 above)
Comment (odow) [reply docs/Project.toml:8]: Oops. Debugging locally.

File: docs/Project.toml (hunk 2 above)
Comment (odow) [reply docs/Project.toml]: Guess not.

**Review by blegat — APPROVED** [review]
